targets/TARGET_WICED/WICED.md

```diff
@@ -0,0 +1,244 @@
+# How to add the WICED SDK
+
+This document summarizes how to download and integrate the WICED SDK into Mbed OS.
+
+**NOTE:** This document is intended for when the WICED SDK is extracted from Mbed OS.
+
+There are several files not provided by the WICED SDK. This is a non-exhaustive
+list of files that are currently provided by Mbed OS:
+- w_generated_mac_address.txt
+- w_generated_security_dct.h
+- w_resources.h
+- mbed_lib.json
+
+## Download the WICED SDK and Advantech patch for the WISE1530
+
+You can find the full instructions to download the WICED SDK with Advantech's patches
+on Advantech's Wiki:  
+http://ess-wiki.advantech.com.tw/view/WISE-1530_SDK
+
+To get the WICED SDK, download WICED-Studio-4.1 for your platform:  
+https://community.cypress.com/community/wiced-wifi/wiced-wifi-documentation
+
+Once downloaded, move the folder `43xxx_Wi-Fi` into `targets/TARGET_WICED`. Rename
+the `43xxx_Wi-Fi` folder to `WICED`.
+
+For the WISE1530, download the patch from Advantech:  
+http://ess-wiki.advantech.com.tw/wiki/images/4/44/WM-BN-BM-22_SDK_5.1.x_platform_patch_v1.5.zip
+
+Once downloaded, copy the contents into `targets/TARGET_WICED/WICED`.
+
+## Enable compilation of the WICED SDK and Mbed OS wrapper
+
+To enable compilation of the WICED SDK, add the label `WICED` to the `target.extra_labels`
+in your mbed_app.json:
+
+``` json
+{
+    "target_overrides": {
```

Comment:
target_overrides isn't working in this case. Should it be:
-        "extra_labels_add": ["STM", "STM32F4", "Advantech_F412RE", "CORDIO"],
+        "extra_labels_add": ["STM", "STM32F4", "Advantech_F412RE", "CORDIO", "WICED"],

Reply:
`"target.extra_labels_add": ["WICED"]` works for me.